tools/collect_debug_info.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+#
+# This script will save debug info to either a gist or /tmp/
+
+# Are we running as root?
+if [[ $EUID -ne 0 ]]; then
+  echo "This script must be run as root. Please re-run like this:"
+  echo
+  echo "sudo $0"
+  echo
+  exit
+fi
+
+echo "This script produces a diagnostic log to help the maintainers"
+echo "figure out why your Mail-in-a-Box installation isn't working the"
+echo "way you expected it to."
+
+echo
+echo "This log will contain sensitive information about your installation"
+echo "including, but not limited to:"
+echo "email addresses"
+echo "domain names"
+echo "IP addresses"
+echo "server security configuration"
+echo "etc."
+echo
+echo "====================================================================="
+echo "Please do not post this file to the internet if you are not comfortable"
+echo "exposing this information publicly to the world forever."
```

Comment:
I know why you put forever there, because nothing ever is really deleted from the internet. Perhaps we should mention somewhere that the gist is secret and only people with the link can view it. Also it can be deleted afterwards. It would be helpful to mention that they can do that at gist.github.com.